main.js

```diff
@@ -25,7 +28,7 @@ inputSeconds.addEventListener("keyup", validateNumberSeconds);
 form.addEventListener("submit", function() {
   event.preventDefault();
 })
-
+timerButton.addEventListener('click', startCountdown);
```

Comment:
This should probably be next to the other button event listeners for readability?